pending-documentation.md

```diff
@@ -107,6 +107,143 @@ argument and parameter are K, and it has no effect on the resulting sort of
 the term. However, it will nevertheless affect the kore generated from the term
 by introducing an additional parameter to the symbol generated for the term.
 
+### `strict` and `seqstrict` attributes
+
+The strictness attributes allow defining evaluation strategies without having to
+explicitely make rules which implement them. This is done by injecting *heating*
+and *cooling* rules for the subterms. For this to work, you need to define what
+a *result* is for K, by extending the  `KResult` sort.
+
+For example:
+
+```k
+syntax AExp ::= Int
+              | AExp "+" AExp [strict]
+```
+
+This generates two heating rules (where the hole syntax `[]` is automatically
```

Comment:
this makes it sound like we are creating a new production `syntax AExp ::= "[]"` but actually it's more like
```
syntax AExp ::= AExp "+" "[]"
syntax AExp ::= "[]" "+" AExp
```

Reply:
Fixed.